lib/puppet/parser/functions/zookeeper_servers_url.rb

```diff
@@ -0,0 +1,27 @@
+module Puppet::Parser::Functions
+  newfunction(:zookeeper_servers_url, :type => :rvalue, :doc => <<-EOS
+This function converts an array of zookeeper hostnames into a combined URL for
+zookeeper HA. You may pass a second argument to indicate the zookeeper port.
```

Comment:
Sure, no problem. I've merged this into [dev](https://github.com/deric/puppet-mesos/compare/v0.7.1...dev). With a backward compatibility fix it should be possible to merge this into master soon, if you don't want any other modifications.